docs/Versions.asciidoc

```diff
@@ -73,3 +73,11 @@ Shared attribute values are pulled from elastic/docs
 ///////
 
 include::{asciidoc-dir}/../../shared/attributes.asciidoc[]
+
+///////
+APM does not build n.x documentation. Links from .x branches should point to master instead
+///////
+ifeval::[{source_branch} == 7.x]
+:apm-server-ref:       {apm-server-ref-m}
+:apm-overview-ref-v:   {apm-overview-ref-m}
+endif::[]
```

Comment:
Should we do something similar if `{source_branch} == master`?

I feel like this will make it easier for contributors to use a single set of attributes that just work. That way we won't
need to remember to use `apm-server-ref-m` in `master` but  `apm-server-ref` in all other branches.

Reply:
The way I set this up, no one should ever have to use `apm-server-ref-m`. All links, regardless of branch, should still use the `apm-overview-ref-v` attribute. The `-m` will only be applied, programmatically, in 7.x. Any other links will still use the `{branch}` attribute to resolove.

Reply:
Gotcha. Thanks for explaining.

Reply:
Cool. In Kib, we originally had it set up like this, which I find a little easier to visualize, but much more difficult to maintain -- hence the attributes replacing attributes in this version.
```
ifeval::[{branch} == 7.x]
:apm-server-ref:       https://www.elastic.co/guide/en/apm/server/master
:apm-overview-ref-v:   https://www.elastic.co/guide/en/apm/get-started/master
endif::[]
```